exporters/common/src/main/java/io/opentelemetry/exporter/internal/grpc/GrpcExporterBuilder.java

```diff
@@ -273,7 +273,7 @@ public String toString(boolean includePrefixAndSuffix) {
       joiner.add("executorService=" + executorService);
     }
     joiner.add("exporterType=" + exporterType.toString());
-    joiner.add("internalTelemetrySchemaVersion=" + internalTelemetryVersion);
+    joiner.add("internalTelemetryVersion=" + internalTelemetryVersion);
```

Comment:
These changes in the exporter are a fix of an oversight from #7265, I simply forgot to fix the names in the `toString()` methods after renaming during review.